assets/images/logo/stark_logo_black.svg

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
```

Comment:
The SVGs of the logos should be placed in `packages/stark-core/assets`... following the same structure as in stark-ui.

Then, you need to add the "assets" entry in the `angular.json` file in `packages/stark-core` to add these logos so they will be copied into the app assets while building the app. See how is done in the `angular.json` file in `packages/stark-ui`